shell/platform/android/io/flutter/embedding/android/FlutterFragment.java

```diff
@@ -405,30 +426,186 @@ private Drawable getSplashScreenDrawableFromActivityTheme() {
     }
   }
 
+  @Override
+  public void onStart() {
+    super.onStart();
+    Log.d(TAG, "onStart()");
+    flutterEngine.getSystemChannels().lifecycle.appIsInactive();
+  }
+
+  public void onPostResume() {
+    Log.d(TAG, "onPostResume()");
+    // TODO(mattcarroll): what does 'updateAccessibilityFeatures' do? why is it called here?
+    flutterView.updateAccessibilityFeatures();
+    // TODO(mattcarroll): why does the platform plugin care about "post resume"?
+    platformPlugin.onPostResume();
+    // TODO(mattcarroll): why does the lifecycle wait for "post resume" to say we're resumed?
```

Comment:
Similar comment on `onResume` vs `onPostResume`.

/cc @jason-simmons for thoughts